vars/buildPlugin.groovy

```diff
@@ -47,6 +49,11 @@ def call(Map params = [:]) {
       label = agentContainerLabel
     }
 
+    // DEBUG: concatenate passed labels to be able to test specific artifact caching providers (ex: " && doks")
+    if (additionalAgentLabels != null && additionalAgentLabels != '') {
+      label += additionalAgentLabels
+    }
```

Comment:
Added for testing artifact caching proxy, but could be useful otherwise, WDYT @dduportal?
Should I open a PR to add it or should I forget about it?

Reply:
I'm not convinced as it would increase the ability to bypass what the infra expects.

Example for "highmem:" there were 5-6 plugins using highmem to solve a "short term" problem in 2019 who never stopped until we cleaned it up.

Since it's only "paranoid and hypothetical", it's not a valid argument from me, but "no is temporary, yes is definitive).

So I would say: "Temporary no because there is no evident use case for users"